grails-app/controllers/au/org/ala/specieslist/SpeciesListController.groovy

```diff
@@ -347,6 +367,12 @@ class SpeciesListController {
             def isdr = params.id.startsWith("dr")
             def guids = getGuidsForList(params.id,grailsApplication.config.downloadLimit)
             def speciesList = isdr?SpeciesList.findByDataResourceUid(params.id):SpeciesList.get(params.id)
+
```

Comment:
`/speciesList/fieldGuide` is invoked when  attempting to download a list and should be invoke-able by a non-authenticated user. This check prevents a non-authenticated user for invoking this method.